processor/samplingprocessor/config.go

```diff
@@ -36,5 +36,9 @@ func (cfg Config) Validate() error {
 		return errInvalidDropRatio
 	}
 
+	if cfg.Condition == "" {
+		cfg.Condition = "true"
+	}
```

Comment:
Do we ever hit this case? We already set the default to this in createDefaultConfig. IMO we shouldn't modify the config in a "Validate" function, I wouldn't expect validation to mutate the config.

Reply:
Good point, I wasn't sure how the condition field would be set if it isn't passed in from the config, but I see now with some testing that it gets set to the default. Change has been pushed 👍